packages/desktop-gui/src/prompts/prompts-model.js

```diff
@@ -0,0 +1,106 @@
+import _ from 'lodash'
+import { action, computed, extendObservable } from 'mobx'
+import interval from 'human-interval'
+
+const prompts = _.sortBy([
+  {
+    slug: 'ci1',
+    interval: interval('4 days'),
+    noProjectId: true,
+  },
+  // {
+  //   slug: 'dashboard1',
+  //   interval: interval('7 days'),
+  //   noProjectId: true,
+  // },
```

Comment:
Can this be removed?

Reply:
I had this commented since we are planning on getting this prompt updated and in for the next release (after this Monday) but I'll remove it and add it all back in a separate PR.